service/src/main/kotlin/no/nav/su/se/bakover/service/søknadsbehandling/SøknadsbehandlingServiceImpl.kt

```diff
@@ -700,11 +701,39 @@ internal class SøknadsbehandlingServiceImpl(
         return oppdatertBehandling.right()
     }
 
-    override fun lukk(lukketSøknadbehandling: LukketSøknadsbehandling, sessionContext: SessionContext) {
-        søknadsbehandlingRepo.lagre(lukketSøknadbehandling, sessionContext)
+    override fun lukk(lukketSøknadbehandling: LukketSøknadsbehandling, tx: TransactionContext) {
+        søknadsbehandlingRepo.lagre(lukketSøknadbehandling, tx)
     }
 
-    override fun lagre(avslag: AvslagManglendeDokumentasjon, sessionContext: SessionContext) {
-        return søknadsbehandlingRepo.lagreAvslagManglendeDokumentasjon(avslag, sessionContext)
+    override fun lagre(avslag: AvslagManglendeDokumentasjon, tx: TransactionContext) {
+        return søknadsbehandlingRepo.lagreAvslagManglendeDokumentasjon(avslag, tx)
+    }
+
+    override fun leggTilOppholdIUtlandet(request: LeggTilOppholdIUtlandetRequest): Either<SøknadsbehandlingService.KunneIkkeLeggeTilOppholdIUtlandet, Søknadsbehandling> {
```

Comment:
```suggestion
    override fun leggTilOppholdIUtlandet(request: LeggTilOppholdIUtlandetRequest): Either<SøknadsbehandlingService.KunneIkkeLeggeTilOppholdIUtlandet, Søknadsbehandling.Vilkårsvurdert> {
```
Her wrapper vi ikke right som i revurdering, så dette bør vel gå?